rdr_service/offline/retention_eligible_import.py

```diff
@@ -108,6 +111,152 @@ def import_retention_eligible_metrics_file(task_data):
     logging.info(f"Import and update completed for gs://{csv_file_cloud_path}")
 
 
+def calculate_retention_eligible_metrics():
```

Comment:
Can you add `app_util`'s `nonprod` decorator to this method, just to make certain it doesn't get triggered in Prod on accident?

Reply:
Good idea, done!